### PR TITLE
feature/security bundle command

### PR DESCRIPTION
**Pull request by lapistano**
Thought it would be useful to have an ability to list every available security authentication key finder strategy.

### CODE REVIEW

File: src/Graviton/SecurityBundle/Command/AuthenticationKeyFinderCommand.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('graviton:security:authenication:keyfinder:strategies')
+            ->setDescription('Greet someone')
```
Comment (hairmare) [inline src/Graviton/SecurityBundle/Command/AuthenticationKeyFinderCommand.php]: s/Greet someone/List strategies/

File: src/Graviton/SecurityBundle/DependencyInjection/Compiler/AuthenticationKeyFinderPass.php [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                        $serviceId,
+                        new Reference($serviceId)
+                    )
+                );
+            } else {
+
+                if ($container->hasDefinition('logger')) {
+                    $logger = $container->getDefinition('logger');
+
+                    $logger->warming(
```
Comment (hairmare) [inline src/Graviton/SecurityBundle/DependencyInjection/Compiler/AuthenticationKeyFinderPass.php]: Shoudn't this be ``warning``?

**Comment by lapistano** [comment]
PR is still under dev... There are Tests missing. Will do it later today.

**Comment by lapistano** [comment]
Todos:
- write tests ...            done
- fix occuring bugs ... done 
- docu ...                    done

**Pull request merged**